dsl_parser/elements/imports.py

```diff
@@ -192,6 +197,9 @@ def location(value):
     imports_graph = ImportsGraph()
     imports_graph.add(location(dsl_location), parsed_dsl_holder)
 
+    def _is_parsed_resource(imported_resource):
+        return isinstance(imported_resource, Holder)
```

Comment:
so the new resolver in cloudify-cosmo/cloudify-manager#1486 returns a Holder?

Reply:
Yes, the yaml